renormalizer/cv/spectra_cv.py

```diff
@@ -49,22 +50,20 @@ def check_qn(self, X):
         print('Quantum number of X', check_1)
     '''
     def run(self):
-        try:
+        if importlib.util.find_spec("cupy"):
```

Comment:
Could you explain why `forkserver` is necessary for `cupy`? I'm sorry but I'm not familiar with this feature.

Reply:
I found the same problem and solution at [pytorch/pytorch/issues/334] and [chainer/chainer/issues/1087], my understanding is that before starting multiprocessing, the cuda has already been initialized, so actually each process did't inherent the cuda resource, that means cuda should be initialized at each process separately by setting the start method as `spwan` or `forkserver`

Reply:
Thanks for the explanation.